src/cljc/i18n/en.cljc

```diff
@@ -746,7 +746,7 @@
           :bad-pub-count (fn [[base additional]] (str base (when (pos? additional) (str " + " additional)) " Bad Publicity"))
           :mu-count (fn [[unused available]] (str unused " of " available " MU unused"))
           :special-mu-count (fn [[unused available mu-type]] (str unused " of " available " " mu-type " MU unused"))
-          :indicate-action "Indicate action"
+          :indicate-paid-ability "Indicate paid ability"
```

Comment:
either don't change the key or change it in all of the translations

Reply:
oh yeah, I completely forgot lmao :joy: yeah I'll change it back next time I get a minute